src/NServiceBus.AcceptanceTests/Timeout/CyclingOutageTimeoutPersister.cs

```diff
@@ -95,7 +95,7 @@ public Task<TimeoutData> Peek(string timeoutId, ContextBag context)
             return Task.FromResult<TimeoutData>(null);
         }
 
-        public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice)
+        public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice, int maxChunkSize = Int32.MaxValue, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
@andreasohlund 

https://github.com/aspnet/EntityFramework/blob/2bc7eaa29cc57f7d550ab4eeba24b07c940595a2/src/Microsoft.EntityFrameworkCore.Relational/Storage/IRelationalDatabaseCreator.cs#L13
https://github.com/aspnet/EntityFramework/blob/2bc7eaa29cc57f7d550ab4eeba24b07c940595a2/src/Microsoft.EntityFrameworkCore.Relational/Migrations/IHistoryRepository.cs
https://github.com/aspnet/EntityFramework/blob/2bc7eaa29cc57f7d550ab4eeba24b07c940595a2/src/Microsoft.EntityFrameworkCore/Storage/IDbContextTransactionManager.cs

This is sorta best practices